pkg/audit/disqualification_test.go

```diff
@@ -49,14 +49,14 @@ func TestDisqualificationTooManyFailedAudits(t *testing.T) {
 		require.NoError(t, err)
 
 		var (
-			sat    = planet.Satellites[0]
-			nodeID = planet.StorageNodes[0].ID()
-			report = &audit.Report{
+			satellitePeer = planet.Satellites[0]
```

Comment:
For the `sat`, `satellite` rename to `satellitePeer`. Either seemed better, because I don't think adding the `Peer` adds any new information to the variable name. `sat` was unique enough, I don't think we have conflicts with other things... I would keep it as `satellite`.

Reply:
in this case `satellite` masquerades a package identifier.
In my opinion `sat` is fine.

We could think to add our business domain abbreviations into a document (kind of a glossary), not all at once but have the document and start to add the ones that we start to use and are clear for all of use without that much discussion.

Reply:
Yeah, the naming conflict in terms of package identifier will be inevitable, since there are so many packages. Of course, having that naming conflict is not ideal. I haven't yet gotten confused with `satellite` package and variable name... however with `uplink` I have. So not quite sure what the best balance is.

Reply:
There are many packages, but normally only a few collide, I don't think that it's very hard to find another name without clashing with a package identifier.

I think that it's hard to convey to conventions where some people see that a specific thing confuse them while not to others, straight and tangible conventions are easier to follow and contribute to be more respected.

For this specific case, `satellitePeer` looks better to me than `satellite` because of the last added convention in our style guide despite it's a long name because it turns that we have quite a few long named variables in our sources and they have never been a problem.